installer-resources/linux/lantern.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
We use this file to bootstrap the Lantern binary in a way that allows the binary to update itself.